sct-commons/src/test/java/org/lfenergy/compas/sct/commons/UtilsTest.java

```diff
@@ -0,0 +1,144 @@
+// SPDX-FileCopyrightText: 2021 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons;
+
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UtilsTest {
+
+    @Test
+    void getField_should_return_declaredField() {
+        // Given : nothing
+        // When
+        Field childField = Utils.getField(ChildClass.class, "childField");
+        // Then
+        assertNotNull(childField);
+        assertEquals("childField", childField.getName());
+    }
+
+    @Test
+    void getField_should_return_declaredField_when_declared_on_superclass() {
+        // Given : nothing
+        // When
+        Field childField = Utils.getField(ChildClass.class, "parentField");
+        // Then
+        assertNotNull(childField);
+        assertEquals("parentField", childField.getName());
+    }
+
+    @Test
+    void getField_should_return_null_when_field_not_found() {
+        // Given : nothing
+        // When
+        Field nonExistentField = Utils.getField(ChildClass.class, "non existent field");
+        // Then
+        assertNull(nonExistentField);
+    }
+
+    @Test
+    void setField_should_set_declaredField() {
+        // Given
+        ChildClass child = new ChildClass();
+        // When
+        Utils.setField(child, "childField", "changed");
+        // Then
+        assertEquals("changed", child.getChildField());
+    }
+
+    @Test
+    void setField_should_set_declaredField_with_value_null() {
+        // Given
+        ChildClass child = new ChildClass();
+        // When
+        Utils.setField(child, "childField", null);
+        // Then
+        assertNull(child.getChildField());
+    }
+
+    @Test
+    void setField_should_set_declaredField_when_declared_on_super_class() {
+        // Given
+        ChildClass child = new ChildClass();
+        // When
+        Utils.setField(child, "parentField", "changed");
+        // Then
+        assertEquals("changed", child.getParentField());
+    }
+
+    @Test
+    void setField_should_do_nothing_when_field_not_found() {
+        // Given
+        ChildClass child = new ChildClass();
+        // When & Then
+        assertDoesNotThrow(() ->
+            Utils.setField(child, "non existent field", "changed"));
+    }
+
+    @Test
+    void setField_should_do_nothing_when_value_is_not_assignable() {
```

Comment:
why value could not be assignable ?

Reply:
Suggest as there is question about test : 
setField_should_do_nothing_when_value_types_mismatch

Reply:
OK. I use the suggestion of @AliouDIAITE.